exonum/src/api/websockets.rs

```diff
@@ -0,0 +1,161 @@
+// Copyright 2018 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! WebSockets
+
+use actix::*;
```

Comment:
Is asterisk really needed here?

Reply:
We can replace it with `use actix::{fut, prelude::*};` (with asterisk too), but I think it's not better.
We need this asterisk because of `actix_derive` and similar uses in actix's lib file.
Correct me if I'm wrong